### ISSUE TITLE
When screen reader is enabled, select-to-copy UI is not disabled

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va.gov-team#25635]
## What happened?
With Talkback enabled on Android, the following occurs:
     - The select-to-copy UI is still enabled (it should be disabled when a screen reader is on)
     - All blocks of text when highlighted to be read with Talkback, announce "double tap to activate, double tap and hold to long press." A sightless user will be confused as to what text should actually have that announcement.

On iOS, the select-to-copy UX is also enabled when VoiceOver is on. 

## Specs:
- Device: Galaxy Note 9, iPhone 11

## Steps to Reproduce
Enable Talkback, read a block of text.

Tips for reproducing on iOS: 
1. Turn on VoiceOver
2. Go to a page that has several ListItems, such as Profile
3. Quickly single tap between some of the list items:
     - Tap Military Info
     - Tap VA Letters and documents
     - Tap back to Military Info
4.As you tap back and forth between the list items, long press and hold down on one until the select-to-copy UX appears
     - The key to triggering this seems to be rapidly single-tapping back and forth, so that the VoiceOver announcement on one item is still being read while you've tapped on another item.

## Desired behavior
Select-to-copy UX should be disabled when screen reader is enabled.

## Acceptance Criteria
1. When the device's screen reader is enabled (Voice Control, Voice Assistant, etc.), this should disable the select-to-copy UX

## Bug Severity
- 3 - Medium

## Linked to Story
#24558 - see Accessibility Implementation Guidelines

## Screen shot(s) and additional information

The linked story (#24558) states the following in the accessibility guidelines:

Accessibility Implementation Guidelines:

When the device's screen reader is enabled (Voice Control, Voice Assistant, etc.), this should disable the select-to-copy UX

This is a shot with Talkback enabled and actually being able to select-to-copy text UX...  This should be "off."

![image.png](https://images.zenhubusercontent.com/5f6a83c84093730d05b5cee6/366062d6-bf49-4a25-8e79-72a43fa26c97)

Screenshot of VoiceOver enabled w/ select-to-copy UX: 
![Image from iOS (23).png](https://images.zenhubusercontent.com/5f64e41291069a617acc1b47/adbbb031-e693-4068-8b31-f09cbb6b2942)

**Comment by bessgreen** [comment]
DF dev is blocked - unable to reproduce issue on a simulator. Dev needs a physical Android device (such as Galaxy Note 9) to verify that a fix works.

**Comment by bessgreen** [comment]
Update: issue also occurs on iOS.

**Comment by digitalken** [comment]
PASS 6/18/2021 - Looks like this is fixed on iOS and Android.  Closing.
